src/Microsoft.Data.Analysis/DataFrameColumn.cs

```diff
@@ -17,6 +17,12 @@ namespace Microsoft.Data.Analysis
     /// </summary>
     public abstract partial class DataFrameColumn : IEnumerable
     {
+        /// <summary>
+        /// The base <see cref="DataFrameColumn"/> constructor.
+        /// </summary>
+        /// <param name="name">The name of this column.</param>
+        /// <param name="length">The length of this column.</param>
+        /// <param name="type">The type of data this column holds.</param>
         public DataFrameColumn(string name, long length, Type type)
```

Comment:
from @eerhardt before:

(existing issue) - since this is an abstract class, the constructor should be marked protected.